test/run_error_tests.py

```
@@ -0,0 +1,124 @@
+import unittest
+from datetime import datetime
+from typing import Dict, List, Any
+
+class MockFileSystem:
+    def __init__(self):
+        self.files = {}
+        self.access_errors = False
+        
+    def set_access_error(self, state: bool):
```

Ruff (0.8.2)

Missing return type annotation for public function `set_access_error`

Add return type annotation: `None`

(ANN201)